test/messages.js

```diff
@@ -260,6 +262,7 @@ tape('properly close if destroy called with a open request', function (t) {
   a.read = b.write.bind(b); b.read = a.write.bind(a)
 
   b.request(7, function (err, value) {
+    t.ok(err, "request received error")
   })
 
   b.destroy(true)
```

Comment:
Test case "destroy sends not more than one message" *below* fails, not this one above ("properly close if destroy called with open request")